### DISCUSSION TITLE
Ion-Induced Heating of the Electrode

### DISCUSSION THREAD

**Discussion by keniley1** [shannon-lab/zapdos#92] [General / Ideas]
Hey all,

I have an undergraduate working on a problem that involves using the heat conduction module. I have two regions, the plasma and the electrode, with a temperature variable solved in each. This works fine. 

What I'm confused about is how I would apply a boundary condition that takes into account electrode surface heating via ion bombardment. We can roughly calculate the energy flux by multiplying the ion charged flux with the recombination energy, which gives us W m^-2. But this is where I get stuck -- by default, the temperature boundary conditions across an interface would be equality of heat flux, e.g. (-k dT/dx)_electrode = (-k dT/dx)_plasma. But if I apply a new heat flux BC on the plasma side to take this ion heating into account, would I no longer be taking heat conduction into account? The heat flux into the cathode surface should be both heat conduction and ion heating together.

Basically, I just want to know if I can just create an ion bombardment heat flux BC and add that, or if I need an InterfaceKernel to allow heat conduction from the cathode into the plasma, and heat conduction PLUS ion bombardment from the plasma into the cathode.

**Comment by lindsayad** [comment]
> by default, the temperature boundary conditions across an interface would be equality of heat flux, e.g. (-k dT/dx)_electrode = (-k dT/dx)_plasma.

If you have two different variables, by default the flux would be zero at the interface, and by default I mean if you do nothing in your input file about it, e.g. no bcs, no interface kernels.

> Basically, I just want to know if I can just create an ion bombardment heat flux BC and add that, or if I need an InterfaceKernel to allow heat conduction from the cathode into the plasma, and heat conduction PLUS ion bombardment from the plasma into the cathode.

It sounds to me like you want both

1. InterfaceKernel that allows a heat conduction flux between the two subdomains
2. An additional integrated boundary condition or interface kernel (the choice depends on whether you need information from both subdomains to form this flux) that adds a flux due to the ion bombardment

These fluxes/residuals will be additive/superimpose.

**Reply by keniley1** [reply]
> If you have two different variables, by default the flux would be zero at the interface, and by default I mean if you do nothing in your input file about it, e.g. no bcs, no interface kernels.

Yeah, I should have clarified that in the current input file this is a single continuous variable called "T". In that case flux continuity would be naturally enforced across an interface since it's the same variable on both sides, correct? I would split it into separate variables to apply an InterfaceKernel, of course.

> An additional integrated boundary condition or interface kernel (the choice depends on whether you need information from both subdomains to form this flux) that adds a flux due to the ion bombardment

Alright, I think I'll take this approach. The ion heating boundary condition only depends on the ion flux so it shouldn't require any information from the electrode subdomain.

**Reply by lindsayad** [reply]
> Yeah, I should have clarified that in the current input file this is a single continuous variable called "T". In that case flux continuity would be naturally enforced across an interface since it's the same variable on both sides, correct?

Well continuous finite elements are not locally conservative generally but yes your concept is right.
